manual/configuration.md

```diff
@@ -271,6 +271,31 @@ AllCops:
 # ...
 ```
 
+**Note**: When inspecting a certain directory(or file)
+given as Rubocop's command line arguments,
```

Comment:
The spelling of the name is `RuboCop` :-)

Reply:
Thank you for point out my mistake!
I corrected it.